cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -540,73 +398,13 @@ void CobTwistController::odometryCallback(const nav_msgs::Odometry::ConstPtr& ms
         return;
     }
 
-    //ROS_INFO("Crossproduct : %f %f %f",res(0),res(1),res(2));
-    //ROS_INFO("TCP: %f %f %f",bl_transform_ct.getOrigin().x(),bl_transform_ct.getOrigin().y(),bl_transform_ct.getOrigin().z());
+    //ROS_INFO("Crossproduct : %f %f %f", res(0), res(1), res(2));
+    //ROS_INFO("TCP: %f %f %f", bl_transform_ct.getOrigin().x(), bl_transform_ct.getOrigin().y(), bl_transform_ct.getOrigin().z());
 
     tf::twistMsgToKDL(msg->twist.twist, twist_odometry_bl);    // Base Twist
 
     // transform into chain_base
-    twist_odometry_transformed_cb = cb_frame_bl * (twist_odometry_bl+tangential_twist_bl);
+    twist_odometry_transformed_cb = cb_frame_bl * (twist_odometry_bl + tangential_twist_bl);
 
     twist_odometry_cb_ = twist_odometry_transformed_cb;
 }
-
-
```

Comment:
re-add the following, if desired...(maybe reduce to most essential parts and cleanup!)